app/controllers/api/posts_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::PostsController < ApplicationController
+  def index
+    posts = Post.all
+    render json: {posts: posts} 
+  end
+end
```

Comment:
I'm still kind of uneasy with this naming convention...

Reply:
Noted, but this is what the team has agreed on!